structure_tests/ext_run.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/sh
+
+command -v docker > /dev/null 2>&1 || { echo "Docker is required to run GCP structure tests, but is not installed on this host."; exit 1; }
```

Comment:
Maybe "docker version" or "docker ps" to verify you can actually connect to the daemon.